src/app/services/fake-goal-management-service.ts

```diff
@@ -0,0 +1,54 @@
+import {Goal} from '../model/goal';
+
+export class FakeGoalManagementService {
+  goal: Goal;
+  goals = new Map<string, Goal>();
+  userGoals = new Map<string, Goal[]>();
+
+  constructor() {}
+
+  setGoal(
+      name: string, startDate: Date, endDate: Date, workload: number,
+      avgWorkload: number) {
+    let id = this.goals.size;
```

Comment:
if variable is not reassigned, prefer using 'const' instead of 'let'

Reply:
soga!